tests/git-ftp-test.sh

```diff
@@ -151,6 +151,24 @@ test_init_fails() {
 	#assertEquals 1 $error_count
 }
 
+test_init_invalid_password() {
+	REMOTE_BASE_URL_DISPLAY="ftp://$GIT_FTP_USER:***@$GIT_FTP_HOST$GIT_FTP_PORT"
+	cd $GIT_PROJECT_PATH
+	init="$($GIT_FTP_CMD init -u $GIT_FTP_USER -p wrongPassword $GIT_FTP_URL 2>&1)"
+	rtrn=$?
+	assertEquals "fatal: Can't access remote '$REMOTE_BASE_URL_DISPLAY'. Failed to log in. Correct user and password? exiting..." "$init"
+	assertEquals 4 $rtrn
```

Comment:
In one of my last commits I copied the error codes over to the test script. We can use them now which should improve the readability of the tests.

Reply:
Yes. I already was wondering why the codes are in the test script but are never used. So I decided to orientate myself on the existing test and do it the same way. I’d say this is a extra pull request since this should be applied to all test, shouldn’t it?

Reply:
Yes, I agree.